charts/sonarqube-lts/templates/sonarqube-sts.yaml

```
@@ -161,7 +161,7 @@ spec:
             {{- . | toYaml | trim | nindent 12 }}
           {{- end }}
       {{- end }}
-      {{- if and .Values.persistence.enabled .Values.initFs.enabeld }}
+      {{- if and .Values.persistence.enabled .Values.initFs.enabled }}
```

In order to keep the previous "faulty" value for backwards compatibility, how about something like (to be adapted):

```suggestion
      {{- if and .Values.persistence.enabled ( or .Values.initFs.enabeld .Values.initFs.enabled ) }}
```